worker.go

```diff
@@ -52,21 +56,47 @@ func waitForTerminationSignals(signals ...os.Signal) chan os.Signal {
 }
 
 func startPolling(quitCH chan struct{}, certsProvider certs.CertificatesProvider, awsAdapter *aws.Adapter, kubeAdapter *kubernetes.Adapter, pollingInterval time.Duration) {
+	items := make(chan *managedItem, maxTargetGroupSupported)
+	go updateStacks(awsAdapter, 1*time.Hour, items)
```

Comment:
I would make the interval configurable via a flag. Default could still be one hour, but it's nice to be able to change this if desired.